gpkit/constraints/tight.py

```diff
@@ -35,7 +35,7 @@ def process_result(self, result):
                         leftsubbed = constraint.left.sub(variables).value
                         rightsubbed = constraint.right.sub(variables).value
             if rel_diff >= self.reltol:
-                msg = ("Constraint [%.30s...] is not tight because "
+                msg = ("Constraint [%.200s...] is not tight because "
                        "the left hand side evaluated to %s but "
```

Comment:
So I see two ways of doing this: 

`"Constraint [%.100s %s %.100s] ..." % (constraint.left, constraint.oper, constraint.right, ...)`

or, more complicatedly, we could print out the variable names without any models that they all share.